README.md

```diff
@@ -1,81 +1,226 @@
 
-### package.json
+To solve this problem, the SuluHeadlessBundle introduces `ContentTypeResolver` services to resolve the content of
+pages to scalar values. The bundle already includes `ContentTypeResolver` services for various content types.
+If your project includes custom content types or if you are not satisfied with an existing `ContentTypeResolver`, 
+you can register your own `ContentTypeResolver` by implementing the `ContentTypeResolverInterface` and
+adding a `sulu_headless.content_type_resolver` tag to the service.
+
+### Provide popular Sulu functionality via JSON APIs
+
+The Sulu content management system comes with various services and Twig extensions to simplify the development and the
+rendering complex websites. This functionality is not available when serving the content of the website in a headless 
+way, therefore the SuluHeadlessBundle includes controllers to **provide JSON APIs for accessing these features**.
+
+- Sulu navigation contexts of the application can be retrieved as JSON object via `{host}/api/navigations/{contextName}`. 
+Similar to the Twig extension, the API respects the following query parameters `depth`, `flat` and `excerpt`.
+
+- The search functionality of SULU is accessible as JSON API via via `{host}/api/search?q={searchTerm}`.
+
+### Reference single page application implementation
+
+The SuluHeadlessBundle is completely **frontend independent** and does not require the use of a specific technology or 
+framework. Still, the bundle contains an **independent and optional single page application setup** in the 
+`Resources/js-website` directory that allows you to quick-start your project and serves as a reference implementation
+for utilizing the bundle functionality. 
+
+The provided reference implementation builds upon **React** as rendering library and utilizes **MobX** for state 
+management. It is built around a central `viewRegistry` singleton that allows you to register React components 
+as view for specific types of resources (eg. pages of a specific template). The application contains a router that will 
+intercept the navigation of the browser, load the JSON data for the requested resource and render the respective view
+with the loaded data. 
+
+![Reference Frontend Implementation](https://user-images.githubusercontent.com/1698337/73056284-f7175100-3e8e-11ea-9e67-9371d8c65099.jpg)
+
+To use the provided single page application setup, you need to include the following lines in your Twig template to
+initialize and start the application:
+
+```twig
+{% block content %}
+    <div id="sulu-headless-container"></div>
+    
+    {# initialize application with json data of current page to prevent second request on first load #}
+    <script>window.SULU_HEADLESS_VIEW_DATA = {{ jsonData|raw }};</script>
+    <script>window.SULU_HEADLESS_API_ENDPOINT = '{{ sulu_content_path('/api') }}';</script>
+    
+    {# start the single page application #}
+    <script src="/build/website/js/index.js"></script
```

Comment:
we need atleast a tag `<body>` around this else if you just copy & paste the example don't work

Reply:
I would use `build/headless`